pkg/login/social/social.go

```diff
@@ -245,13 +286,13 @@ var GetOAuthProviders = func(cfg *setting.Cfg) map[string]bool {
 	return result
 }
 
-func GetOAuthHttpClient(name string) (*http.Client, error) {
-	if setting.OAuthService == nil {
+func (ss *SocialService) GetOAuthHttpClient(name string) (*http.Client, error) {
+	if ss.oAuthProvider[name] == nil {
```

Comment:
Think this if-statement should be removed. Received an error when using "Forward OAuth Identify" for a data source reading "failed to get OAuth http client, error=OAuth not enabled".

Reply:
I removed it